libraries/Microsoft.Bot.Builder.AI.QnA/Models/QnAMakerTraceInfo.cs

```diff
@@ -111,5 +103,9 @@ public class QnAMakerTraceInfo
         /// </value>
         [JsonProperty("rankerType")]
         public string RankerType { get; set; }
+        
+        [Obsolete]
```

Comment:
>Obsolete [](start = 9, length = 8)

Obsolete(...messsage..) 

You  should add a message that describes that this property is no longer used.